QP_solver/doc/QP_solver/CGAL/QP_solution.h

```diff
@@ -495,9 +495,9 @@ then \f$\lambda_i\geq 0\f$ (\f$\lambda_i\leq 0\f$, respectively).
 <LI>
 \f[
 \begin{array}{llll}
-&&\geq 0 & \mbox{if \f$u_j=\infty\f$} \\
+&&\geq 0 & \mbox{if $u_j=\infty$} \\
```

Comment:
Is the issue with `\f$` being the usage inside the array?

Reply:
The `\f$` is a doxygen command so unknown to LaTeX.

Reply:
Of course you are right, we are in a `\f[` env.